File: recypro/src/main/java/com/yema/recypro/test/ReplayMultipleItem.java
package com.yema.recypro.test;

public class ReplayMultipleItem {



    public static int NORME = 1;
    public static int TOMMAIN = 1;

    private int itemType;
    private String content;

    private String responde;

    private String replied;


    public ReplayMultipleItem(int itemType, String content, String responde, String replied) {
        this.itemType = itemType;
        this.content = content;
        this.responde = responde;
        this.replied = replied;
    }

    public int getItemType() {
        return itemType;
    }

    public void setItemType(int itemType) {
        this.itemType = itemType;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public String getResponde() {
        return responde;
    }

    public void setResponde(String responde) {
        this.responde = responde;
    }

    public String getReplied() {
        return replied;
    }

    public void setReplied(String replied) {
        this.replied = replied;
    }
}

File: app/src/main/java/com/yema/mytestlibrary/adapter/ReplayMultipleItem.kt
package com.yema.mytestlibrary.adapter

import com.chad.library.adapter.base.entity.MultiItemEntity

/**
 *  itemType : 类型
 *  content  : 内容
 *  responde : 回复者
 *  replied : 被回复者
 *
 */
class ReplayMultipleItem(private val itemType: Int,  val content: String,  val responde: String,  val replied: String) : MultiItemEntity {
    override fun getItemType(): Int {
        return itemType
    }

    companion object {
        @JvmField
        val NORME = 1
        @JvmField
        val TOMMAIN = 2
    }

}